README.md

```diff
@@ -1,6 +1,6 @@
-# Okta Angular + Custom Login Example
+# Okta Angular + Okta Hosted Login Example
```

Comment:
We should consider changing the name to "Okta Angular + Okta Redirect Model Example". Chris Mills checked with Product and that's the name for OIE and from what I understand, everything is being updated with that naming for consistency.

Reply:
I agree, but I'm a little fuzzy about how the official sample/quickstart samples will work with our new updates. I used the name "Redirect Model" in the quickstart repo I created.

https://github.com/oktadev/okta-angular-quickstart